blaze-server/src/test/scala/org/http4s/server/blaze/Http1ServerStageSpec.scala

```diff
@@ -31,19 +31,18 @@ import org.http4s.blaze.util.TickWheelExecutor
 import org.http4s.blazecore.{ResponseParser, SeqTestHead}
 import org.http4s.dsl.io._
 import org.http4s.headers.{Date, `Content-Length`, `Transfer-Encoding`}
-import org.specs2.specification.AfterAll
-import org.specs2.specification.core.Fragment
+import org.http4s.syntax.all._
 import scala.concurrent.duration._
-import scala.concurrent.Await
 import _root_.io.chrisdavenport.vault._
 import org.http4s.testing.ErrorReporting._
 
-class Http1ServerStageSpec extends Http4sSpec with AfterAll {
-  sequential
+class Http1ServerStageSpec extends Http4sSuite {
+  override def munitFlakyOK = sys.env.get("CI").isDefined
```

Comment:
this is an alternative to the `skipOnCi` we could even move it to `Http4sSuite`, wdyt?

Reply:
:+1: